lib/imports/api/authority/server/authority-manager.js

```diff
@@ -54,15 +51,17 @@ export class AuthorityManager {
       'ProseMeteor.latestDocState' ({ docId }) {
         check(docId, String);
         const connection = this.connection;
-        return self.latestDocState({ docId, connection });
+        const userId = Meteor.userId();
```

Comment:
userId is passed into `latestDocState`, `stepsSince`, and `receiveSteps` to authenticate the "edit" action